app/controllers/v0/profile/communication_preferences_controller.rb

```diff
@@ -7,7 +7,6 @@ module Profile
     class CommunicationPreferencesController < ApplicationController
       service_tag 'profile'
       before_action { authorize :vet360, :access? }
-      before_action { authorize :communication_preferences, :access? }
```

Comment:
Forgotten flipper! [communication_preferences](https://api.va.gov/flipper/features/communication_preferences) flipper has been removed so this is no longer needed.